proposals/0273-webengine-projection-mode.md

```diff
@@ -39,9 +39,9 @@ After pop-up is closed, touch events processing returns back to the WEB_VIEW are
 2. The HMI should include a button to access the app list
 3. The HMI should not show a menu button on the HMI. Instead the app should be required to behave as followed:
 
-    3.1 . If the app uses AddCommand or AddSubMenu the app is required to provide an own menu button on the WEB_VIEW and send ShowAppMenu if this menu button is selected by the user.
+    3.1 . If the app uses `AddCommand` or `AddSubMenu`, the app is required to provide an own menu button on the WEB_VIEW and send `ShowAppMenu` if this menu button is selected by the user.

```

Comment:
```suggestion
    3.1 . If the app uses `AddCommand` or `AddSubMenu`, the app is required to provide an own menu button on the `WEB_VIEW` and send `ShowAppMenu` if this menu button is selected by the user.
```

Reply:
@theresalech Thank you. Done in 01552abc88117f2a183687c20cf622d79eefe484